db/fixtures/server_roles.csv

```diff
@@ -17,5 +17,5 @@ scheduler,Scheduler,1,false,region
 smartproxy,SmartProxy,0,false,zone
 smartstate,SmartState Analysis,0,false,zone
 user_interface,User Interface,0,false,region
-websocket,Websocket,0,false,region
```

Comment:
This fixture gets seeded at boot into the server_roles table.  I don't know if we'd have to drop the existing websocket row from that table.  

It might be fine without dropping the role.  I don't know if we present the existing server roles rows in the UI.

Reply:
Well, as we did the migrations, I think this can just go.